openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -77,12 +90,14 @@ class OrganizationFactory(DictFactoryBase):
     key = factory.Faker('word')
     name = factory.Faker('company')
     uuid = factory.Faker('uuid4')
+    logo_image_url = factory.Faker('image_url')
 
 
 class SeatFactory(DictFactoryBase):
-    type = factory.Faker('word')
-    price = factory.Faker('random_int')
     currency = 'USD'
+    price = factory.Faker('random_int')
+    sku = factory.LazyFunction(generate_seat_sku)
+    type = factory.Faker('word')
```

Comment:
setting it to 'verified' seems enough. no other test is actually doing anything with that value and those tests that really need this value to be something specific should explicitly set this value for readability reasons.